clifford/_multivector.py

```diff
@@ -773,6 +773,9 @@ def inv(self) -> 'MultiVector':
     leftInv = leftLaInv
     rightInv = leftLaInv
 
+    def pinv(self) -> 'MultiVector':
+        return self._newMV(self.layout.pinv_func(self.value))
```

Comment:
Mind moving the docstring to here instead, since this is likely where it will be seen?